framework/src/media/MediaRecorderImpl.cpp

```diff
@@ -313,6 +313,7 @@ void MediaRecorderImpl::stopRecorder(recorder_result_t ret)
 	}
 
 	mOutputHandler.flush();
+	notifyObserver(RECORDER_OBSERVER_COMMAND_FLUSH);
```

Comment:
We'd better stop device flush.
Stop means no more recording, we can sync it in notifyObserver when cmd is RECORDER_OBSERVER_COMMAND_FINISHIED or RECORDER_OBSERVER_COMMAND_STOP_ERROR

Reply:
And please consider sync for all error case, double checking needed.

Reply:
thanks @Taejun-Kwon ,
i will close this commit.

Reply:
thanks @Taejun-Kwon ,
i will close this commit.